### PR TITLE
adjust some Blitz API responses to signal "OK"

### PR DESCRIPTION
**Pull request by mtbc**
It looked to me as if these Response objects clearly indicate success thus should extend "OK". Happy to revert any for which there is doubt.

### CODE REVIEW

**Comment by joshmoore** [comment]
Interesting. Can you point me to the consuming code where you were seeing this being beneficial? I see the intent, but I don't know if we make any assumptions anywhere about the difference between `OK` and a concrete `Response`. (Likely not, just trying to think it through)

**Comment by mtbc** [comment]
I'm afraid I forget the concrete example but it was probably in some OMERO.cli plugin where there were conditionals and a mention of ERR and I wanted to write OK in the code to clearly contrast with the other then was surprised that the response I was hoping to pass on didn't actually count as being OK. (It could also have been my graphs request testing script given that `DiskUsage2` is one of those changes: I might have just wanted to print the response or whatever if OK.) It depends partly on what "OK" is really meant to mean but I figured those responses probably count (or maybe we should just drop OK). I suspect I occasionally want use it as a catchall partly where there are alternative responses, like it could be either `UsedFilesResponse` or `UsedFilesResponsePreFs` but if it's OK then at least I know it's not ERR.

**Comment by joshmoore** [comment]
Understood. I don't see dropping it, and your reasoning is sound. Just requires a bit of review. It's likely a good time as we move towards more web-based APIs to think about how we move from `OK` and `ERR` to something more like the http status codes.

File: src/main/slice/omero/cmd/FS.ice [hunk 1]
```diff
@@ -76,7 +76,7 @@ module omero {
          * will be returned under normal conditions, otherwise a {@link ERR}
          * will be returned.
          **/
-        class OriginalMetadataRequest extends Request {
+        class OriginalMetadataRequest extends OK {
```
Comment (jburel) [inline src/main/slice/omero/cmd/FS.ice]: This should be reverted. Training failed https://web-proxy.openmicroscopy.org/west-ci/job/OMERO-training/335

**Comment by joshmoore** [comment]
This has been in for a while and I haven't seen any unintended consequences. The largest impact I can _imagine_ is that 5.4/5.5 communication could get weird, but that isn't something we want to support anyway. Merging.

**Pull request merged**